hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationServer.java

```diff
@@ -218,6 +225,15 @@ public static final class ReplicationConfig {
     )
     private double outOfServiceFactor = OUTOFSERVICE_FACTOR_DEFAULT;
 
+    @Config(key = ZEROCOPY_ENABLE_KEY,
+        type = ConfigType.BOOLEAN,
+        defaultValue =  ZEROCOPY_ENABLE_DEFAULT_VALUE,
+        tags = {DATANODE, SCM},
+        description = "Specify if zero-copy should be enabled for " +
+            "replication protocol."
+    )
+    private boolean zeroCopyEnable = ZEROCOPY_ENABLE_DEFAULT;
+
```

Comment:
@guohao-rosicky , thanks a lot for working this!

Why would anyone want to disable it?  Let's don't add this conf and enable it always?